apps/fz_http/lib/fz_http_web/router.ex

```diff
@@ -97,6 +101,11 @@ defmodule FzHttpWeb.Router do
     ) do
       live "/user_devices", DeviceLive.Unprivileged.Index, :index
       live "/user_devices/new", DeviceLive.Unprivileged.Index, :new
+
+      live "/user_devices/change_email_or_password",
```

Comment:
Maybe this could be something like `/user_account/change_password` to be consistent with prefixing unprivileged routes with `/user_`? Should probably refactor the prefix convention at some point to `/admin` and `/user`.